okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -270,6 +274,29 @@ SSLSocketFactory createSocketFactory() {
     }
   }
 
+  private static SSLContext getSslContext() throws NoSuchAlgorithmException {
+    if (GrpcUtil.IS_RESTRICTED_APPENGINE) {
+      try {
+        SSLContext tlsContext = SSLContext.getInstance("TLS", Platform.get().getProvider());
+        KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+        keyStore.load(null, null);
```

Comment:
@ejona86 @nmittler Eric / Nathan - are you saying w/ this comment that you have a conscrypt version somewhere already?

I'm gathering that if this were done, we'd get gRPC running on GAE Standard.  So, I'm looking at fixing your comments and making this work this week.  If that's not required, I'd like to know before I start.